common/source/docs/common-scripting-step-by-step.rst

```diff
@@ -5,21 +5,35 @@
 Script Setup and Use Examples
 =============================
 
-Testing a Script in SITL
+Running a Script in SITL
 ========================
 In Linux:
 
-#. Assuming you have setup the build and simulation environment, run the simulation. See :ref:`using-sitl-for-ardupilot-testing`. For example:
+#. Assuming you have setup the build and SITL environment, run SITL. See :ref:`using-sitl-for-ardupilot-testing`. For example:
 
-.. code::
+.. code-block:: bash
 
-    sim_vehicle.py -v plane -f quadplane
+    ./Tools/autotest/sim_vehicle.py -v plane --console
```

Comment:
probably need to talk about pwd...the above and following assumes you are starting SITL from the ardupilot clone directory

Reply:
You got it. I added some details in the initial SITL tutorial too where that was introduced. The wiki was making an assuming that `Tools/autotest` was on PATH.